lib/schema/args/filter.js

```diff
@@ -56,26 +56,27 @@ module.exports = cache => {
   }
 
   const _generateScalarFilter = gqlType => {
+    const numericOperators = Object.values({ ...RELATIONAL_OPERATORS, ...LOGICAL_OPERATORS })
     const filterType = {
       // REVISIT: which filters for binary
       [GraphQLBinary.name]: _generateFilterType(GraphQLBinary, [RELATIONAL_OPERATORS.eq, RELATIONAL_OPERATORS.ne]),
       [GraphQLBoolean.name]: _generateFilterType(GraphQLBoolean, [RELATIONAL_OPERATORS.eq, RELATIONAL_OPERATORS.ne]),
-      [GraphQLDate.name]: _generateFilterType(GraphQLDate, Object.values(RELATIONAL_OPERATORS)),
-      [GraphQLDateTime.name]: _generateFilterType(GraphQLDateTime, Object.values(RELATIONAL_OPERATORS)),
-      [GraphQLDecimal.name]: _generateFilterType(GraphQLDecimal, Object.values(RELATIONAL_OPERATORS)),
+      [GraphQLDate.name]: _generateFilterType(GraphQLDate, numericOperators),
+      [GraphQLDateTime.name]: _generateFilterType(GraphQLDateTime, numericOperators),
+      [GraphQLDecimal.name]: _generateFilterType(GraphQLDecimal, numericOperators),
       // REVISIT: should 'eq'/'ne' be generated since exact comparisons could be difficult due to floating point errors?
```

Comment:
Update comment for exact floating point number matches with `in` operator
```suggestion
      // REVISIT: should 'eq'/'ne'/'in' be generated since exact comparisons could be difficult due to floating point errors?
```